com.unity.shadergraph/Editor/Data/Nodes/Utility/SubGraphNode.cs

```diff
@@ -183,16 +185,19 @@ public override bool allowedInSubGraph
 
         public override bool canSetPrecision
         {
-            get { return false; }
+            get { return asset.subGraphGraphPrecision == GraphPrecision.Graph; }
```

Comment:
We allow subgraph nodes to override precision now, as long as their asset is switchable